src/app/email-validator/email.validator.ts

```diff
@@ -1,17 +1,16 @@
-import {Control} from '@angular/common';
-interface IValidationResult {
-  [key: string]: boolean;
-}
+import {FormControl} from "@angular/forms";
 
 export class EmailValidator {
-  static validEmail(control: Control): IValidationResult {
```

Comment:
@see http://blog.thoughtram.io/angular/2016/03/14/custom-validators-in-angular-2.html